### PR TITLE
Enhance Dropdown native with values and title

### PR DESCRIPTION
**Pull request by pavestru**
Related to #13 and partly #376

Changes proposed in this pull request:

 - Support value attribute inside item object of items array attribute for dropdown
 - Use value attribute (point above) as value inside `<option />` of native version of dropdown
 - [Experimental] Title in native dropdown - use `<option disabled hidden selected>${title}</option>`

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update
 
 # How Has This Been Tested?

So far only manual browser tests were done on
 - Firefox 60 (Mac & Windows)
 - Chrome 66 (Mac & Windows)
 - IE 11 
 
 # Checklist:
 
 - [ ] My code follows the style guidelines of this project
 - [x] I have performed a self-review of my own code
 - [ ] I have commented my code, particularly in hard-to-understand areas
 - [ ] I have made corresponding changes to the documentation
 - [x] My changes generate no new warnings
 - [ ] I have added tests that prove my fix is effective or that my feature works
 - [ ] New and existing unit tests pass locally with my changes
 - [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/components/m-dropdown/_template.js [hunk 1]
```diff
@@ -4,13 +4,18 @@ import classnames from 'classnames';
 
 const arrowIcon = '<axa-icon icon="angle-bracket-down" classes="m-dropdown__icon"></axa-icon>';
 
-const nativeSelect = ({ items, size }) => html`<div class="${classnames('m-dropdown__select-wrap', {
+const nativeSelect = ({ title, items, size, value }) => html`<div class="${classnames('m-dropdown__select-wrap', {
```
Comment (AndyOGo) [inline src/components/m-dropdown/_template.js]: nice 👍

File: dist/components/m-dropdown/index.js [hunk 2]
```diff
@@ -1 +1 @@
...
...teElement("style"),o=document.createTextNode(e);n.appendChild(o),n.setAttribute("data-c-name",t.toLowerCase()),document.querySelector("head").appendChild(n),me[e]=!0}}}]),t}();var ge=/iPad|iPhone|iPod/.test(navigator.userAgent)&&!window.MSStream;function be(n,e,o){var t,r=3<arguments.length&&void 0!==arguments[3]?arguments[3]:{},i=r.capture,a=void 0===i||i,s=r.passive,l=void 0===s||s,d=n.ownerDocument.documentElement,c=Y(d,e,function(e){var t=e.target;if(!n.contains(t)&&n!==t)return o(e)},{capture:a,passive:l});return t=d,ge&&(t.style.cursor="pointer"),c}var ye=function(){for(var e=arguments.length,t=Array(e),n=0;n<e;n++)t[n]=arguments[n];for(var o=t.length,r={},i=0;i<o;++i){var a=t[i];r[a.toUpperCase()]=a}return r}("click","keyup","enter","move","leave","Escape","Esc"),we="data-prevent-default",xe=function(){function o(e){var i=this,t=1<arguments.length&&void 0!==arguments[1]?arguments[1]:{};c(this,o),this._handleClick=function(e,t){i.shouldPreventDefault(t||e.currentTarget)&&e.preventDefault();var n=!i._lastToggleNode,o=t!==i._lastToggleNode,r=!n&&!o;n?(i._notify(ye.ENTER,t),i._onInteractive()):o&&i._notify(ye.MOVE,t,i._lastToggleNode),i._lastToggleNode=t,r&&i._options.sameClickC
```
Comment (AndyOGo) [inline dist/components/m-dropdown/index.js]: @pavestru 
May I asked you to remove generated files.
We use `npm run release`, which takes care of generating `dist/` and `docs/` etc.

**Comment by LucaMele** [comment]
AWESOME pull request, thank you very much! I will add that code snipped in the readme as well in the how to use section where react is.

**Comment by pavestru** [comment]
- added css fallback for hidden option
- removed built artifacts

**Comment by AndyOGo** [comment]
very nice. thanks man 👍

**Pull request merged**